convert/jenkinsjson/covert.go

```diff
@@ -307,6 +307,7 @@ func collectStepsWithID(currentNode jenkinsjson.Node, stepWithIDList *[]StepWith
 
 	if len(currentNode.AttributesMap) == 0 {
 		for _, child := range currentNode.Children {
+			mergeRunSteps(stepWithIDList)
```

Comment:
do you need these? I think we might just want the one in the beginning of the function